docs/packaging_policy.md

```diff
@@ -1,55 +1,102 @@
 # Packaging Policy
 
-This document gathers all the relevant information regarding the general lines to follow while creating new recipes that will eventually be
-part of this repository.
+This document gathers all the relevant information regarding the general lines to follow while creating new recipes that will eventually be part of this repository.
 
 <!-- toc -->
 ## Contents
 
-  * [Sources](#sources)
-  * [Settings](#settings)
-  * [Options](#options)<!-- endToc -->
-
+* [Sources](#sources)
+* [Dependencies](#dependencies)
+* [Build](#build)
+* [Package](#package)
+* [Settings](#settings)
+* [Options](#options)<!-- endToc -->
```

Comment:
You don't need to modify TOC in the PR, a GH action will take care of it after the PR is merged.

Reply:
Ok, my linter was complaining of the formatting.